framework/src/media/voice/SpeechDetector.cpp

```diff
@@ -61,24 +61,23 @@ bool SpeechDetectorImpl::initKeywordDetect(uint32_t samprate, uint8_t channels)
 		return false;
 	}
 
-	int card = -1;
-	int device = -1;
+	int sd_card = -1;
```

Comment:
Let's define INVALID_HW_NUMBER or NONE_HW_NUMBER something else. 
To improve code understanding.

Reply:
I'll replace with INVALID_HW_NUMBER